test_suites/h265/JCT-VC-3D-HEVC.json

```diff
@@ -0,0 +1,223 @@
+{
+    "name": "JCT-VC-3D-HEVC",
+    "codec": "H.265",
+    "description": "JCT-VC 3D-HEVC 3D Extension",
+    "test_vectors": [
+        {
+            "name": "3DHC_C_A_HHI_3",
+            "source": "https://www.itu.int/wftp3/av-arch/jctvc-site/bitstream_exchange/draft_conformance/3D-HEVC/3DHC_C_A_HHI_3.zip",
+            "source_checksum": "b29d7eb608ded5dee9f90345aa0f6d5a",
+            "input_file": "3DHC_C_A.bit",
+            "output_format": "yuv420p",
+            "result": "0069c977b00a3e4d4ea913df50029bbc"
+        },
+        {
+            "name": "3DHC_C_B_HHI_3",
+            "source": "https://www.itu.int/wftp3/av-arch/jctvc-site/bitstream_exchange/draft_conformance/3D-HEVC/3DHC_C_B_HHI_3.zip",
+            "source_checksum": "1764026920593c98be12107d6fb42118",
+            "input_file": "3DHC_C_B.bit",
+            "output_format": "yuv420p",
+            "result": "fa1fe7e370d2d7f181853f9bc8a1a96b"
+        },
+        {
+            "name": "3DHC_C_C_Sharp_3",
+            "source": "https://www.itu.int/wftp3/av-arch/jctvc-site/bitstream_exchange/draft_conformance/3D-HEVC/3DHC_C_C_Sharp_3.zip",
+            "source_checksum": "fe1ddf915cb3301763ea73b7ce2da9c0",
+            "input_file": "3DHC_C_C_Sharp_3/3DHC_C_C.bit",
```

Comment:
I think the folder name should not be part of the input file. I reviewed the .zip and it makes no sense to me